library/src/main/kotlin/me/thanel/swipeactionview/SwipeActionView.kt

```diff
@@ -760,6 +771,39 @@ class SwipeActionView : FrameLayout {
         else -> false
     }
 
+    /**
+     * Tell whether the user has swiped the view far enough to perform a swipe callback
+     * for the first element in a MultiSwipeActionView
+     *
+     * @param swipeDistance The performed swipe distance.
+     *
+     * @return Whether the user has swiped far enough
+     */
+    private fun hasSwipedFarEnoughMultiContainerFirstElement(swipeDistance: Float): Boolean {
```

Comment:
We can get rid of the maxSwipe checks from this function and only check the halfway condition. Then the code which uses this function could be adjusted to correctly work with new implementation (see other comments).

That way we'll be sure that there is no gap between halfway and full swipe which won't react to any actions.